### DISCUSSION TITLE
Register identification BAD

### DISCUSSION THREAD

**Discussion by techdevai-es** [community/community#139007] [GitHub Education]
### Hi everyone,

Last year, even though I was a student and had to provide my student ID and an official certificate proving it, it was impossible to access the student package, since Github insisted that my school (which appeared in the registration database) gave corporate email to students, when it wasn't true. So it was impossible to convince them otherwise.

This year I changed schools: I'm still a student, Github has my school listed, I have a corporate email, I have an ID and I have a certificate. And what's the problem now? Well, I can't register, because I'm told that "you are far from your school," when I'm registering from the same building.

And now what? What do I have to do?

This is frustrating, I've been a student of application programming for over a year, and it's no use to me. Even though I've always had official documentation proving it.

**Comment by alx-mp4** [comment]
Hi @AntiKarma23, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
